regression/goto-cc-cbmc/check-fps-removed/test.desc

```diff
@@ -0,0 +1,5 @@
+CORE
+main.c
+--show-goto-functions
+^EXIT=0$
+^SIGNAL=0$
```

Comment:
What is this actually testing for?

Reply:
As per commit message:
These tests ensure per-function and module passes are run on functions loaded from goto binaries as they are not run by goto-cc

Reply:
How does that particular test ensure this? AFAIK `--show-goto-functions` would work just fine in presence of function pointers etc. (I can see that the other test does what it's supposed to do.)

Reply:
This one in particular checks that function pointers are removed (if they're not then show-goto-functions will cause an error)

Reply:
Why would show-goto-functions require function pointer removal? That would be a bug in show-goto-functions.

Reply:
Just realised that since Daniel asked us to back lazy loading out of CBMC these tests no longer relate to this PR. I'll take them out and fix them up for a separate PR.